torch/_ops.py

```diff
@@ -1098,14 +1103,22 @@ def _has_script_object_arg(schema: torch.FunctionSchema) -> bool:
 # OpOverloadPacket class contains pointer to a base unresolved operator that doesn't correspond to a specific operator
 # You can obtain an OpOverload object through attribute query.
 class OpOverloadPacket:
-    def __init__(self, qualified_op_name, op_name, op, overload_names):
+    __file__ = "torch.ops"
```

Comment:
This technically a class variable FYI, best to annotate it as such with typing.ClassVar for typing linters.

Reply:
@Skylion007 I attempted to do that, but `mypy` actually barfed on it, saying that the base class (`object`, I assume?) has it typed as a `Literal`. That seemed like a weird type to explicitly apply, so I left it alone.

Reply:
Odd, is it possible that might have been from the `_OpNamespace` type below. It also has a `__file__` but inherits from `typing.ModuleType` which does lead to an error with `mypy` because it is overriding an instance variable as a class var. 
![image](https://github.com/user-attachments/assets/7394f5cb-6f85-4e77-8ab9-f32f29e97e73)

I don't see how `__file__` could be inheriting a conflicting annotation from the base type here.

Reply:
Oh, good point, I may have gotten those confused while typing. I'll give it a shot!

Reply:
Done!